client/login/magic-login/emailed-login-link-expired.jsx

```diff
@@ -17,10 +17,17 @@ class EmailedLoginLinkExpired extends Component {
 		hideMagicLoginRequestForm: PropTypes.func.isRequired,
 		recordPageView: PropTypes.func.isRequired,
 		translate: PropTypes.func.isRequired,
+		isGravPoweredClient: PropTypes.bool.isRequired,
 	};
 
 	componentDidMount() {
 		this.props.recordPageView( '/log-in/link/use', 'Login > Link > Expired' );
+
+		if ( this.props.isGravPoweredClient ) {
+			document
+				.querySelector( '.is-grav-powered-login-page' )
+				?.classList.remove( 'is-grav-powered-login-page' );
+		}
```

Comment:
Is there any special reason we want the "link-expired" screen to look like previously? Wouldn't the new one make a better experience?

Reply:
> Is there any special reason we want the "link-expired" screen to look like previously? Wouldn't the new one make a better experience? 

I think we will customize the expired link view via this issue: 105859-Automattic/gravatar, by another PR

> I am not fully sure I understand why are we adding/removing these ".is-grav-powered-login-page" like this. Would you mind explaining it a bit?

There's no WPCC's `client_id` URL parameter on the page. So [the way we get the `oauth2Client` data](https://github.com/Automattic/wp-calypso/pull/82049/files#diff-548031d790bc68869d0fb30e0fd46e08effdb1e788d808909a3e776a2104ac04R253-R256) is different from [the `logged-out` component](https://github.com/Automattic/wp-calypso/blob/38ca194f4c2023d800e5fd775cec18edb015a551/client/layout/logged-out.jsx#L258-L261). So, what we have done here is we get the desired `oauth2Client` and then dynamically hook the `.is-grav-powered-login-page` into the layout element to reuse the style.